project.yaml

```diff
@@ -1382,10 +1392,21 @@ Layer:
             way,
             place,
             name,
-            ref
+            ref,
+            CASE
+              WHEN (population ~ '^[0-9]+$') THEN CAST(population AS integer)
+              WHEN (place = 'city') THEN 100000
+              ELSE 10000
+            END AS population,
```

Comment:
Maybe use INT_population here or something like that, otherwise it's very easy to get confused.